resources/lang/pl_PL/models.yml

```diff
@@ -0,0 +1,57 @@
+generic:
+  createdAt: "utworzono"
+  modifiedAt: "zmodyfikowano"
+  name: "nazwa"
+  email: "adres e-mail"
+  createdBy: "utworzono przez"
+dictionary:
+  _name: "słownik"
+  _names: "słowniki"
+  name: "$t(models.generic.name)"
+  facility: "placówka, właściciel słownika"
+  isFixed: "nieusuwalny / systemowy"
+  positions: "pozycje"
+position:
+  _name: "pozycja słownika"
+  _names: "pozycje słownika"
+  name: "$t(models.generic.name)"
+  dictionary: "słownik"
+  facility: "placówka, właściciel pozycji"
+  isFixed: "nieusuwalna / systemowa"
+  defaultOrder: "domyślna kolejność"
+facility:
+  _name: "placówka"
+  _names: "placówki"
+  name: "$t(models.generic.name)"
+  url: "adres url wewnątrz Memo"
+  members: "osoby przypisane do placówki"
+member:
+  _name: "powiązanie osoby i placówki"
+  _names: "powiązania osób i placówek"
+  user: "użytkownik"
+  facility: "placówka"
+  hasFacilityAdmin: "administrator placówki"
+permissions:
+  _name: "uprawienia"
+  unverified: "niezweryfikowany e-mail"
+  verified: "zweryfikowany e-mail"
+  globalAdmin: "globalny administrator"
+  facilityMember: "użytkownik placówki"
+  facilityClient: "klient placówki"
+  facilityStaff: "pracownik placówki"
+  facilityAdmin: "administrator placówki"
+user:
+  _name: "użytkownik"
+  _names: "użytkownicy"
+  email: "$t(models.generic.email)"
+  createdAt: "$t(models.generic.createdAt)"
+  modifiedAt: "$t(models.generic.modifiedAt)"
+  createdBy: "$t(models.generic.createdBy)"
+  name: "imię i nazwisko"
+  hasGlobalAdmin: "globalny administrator"
+  lastLoginFacility: "ostania używana placówka"
+  passwordExpireAt: "data wygaśnięcia hasła"
+  hasPassword: "ustawiono hasło"
+  password: "hasło"
+  hasEmailVerified: "zweryfikowany e-mail"
```

Comment:
Bardziej może "e-mail zweryfikowany", analogicznie w permissions.

Reply:
tutaj jest istotne raczej żeby były odpowiednie kody tłumaczeń, a wartości to se możecie zmieniać jak bedzie głupio wyglądało albo coś